src/test/tpc/tpch_db_generator_test.cpp

```diff
@@ -58,12 +26,20 @@ TEST(TPCHTableGeneratorTest, TableContents) {
   EXPECT_TABLE_EQ_ORDERED(table_info_by_name.at("nation").table, load_table(dir_001 + "nation.tbl", chunk_size));
   EXPECT_TABLE_EQ_ORDERED(table_info_by_name.at("region").table, load_table(dir_001 + "region.tbl", chunk_size));
 
+#if defined(__has_feature)
+#if (__has_feature(thread_sanitizer) || __has_feature(address_sanitizer))
+  // We verified thread and address safety above. As this is quite expensive to sanitize, don't perform the following
+  // check - double parantheses mark the code as explicitly dead.
+  if ((true)) return;
```

Comment:
I don't understand. We want to break here in case sanitization is turned on? Why can't we just have a `return;`, why do we need explicitly dead code?

Reply:
Any action required?

Reply:
No.